pkgs/development/libraries/quarto/default.nix

```diff
@@ -0,0 +1,55 @@
+{ callPackage
+, lib
+, pandoc
+, esbuild
+, deno
+, stdenv
+, fetchurl
+, nixos
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "quarto";
+  version = "1.0.38";
+
+  src = fetchurl {
+    url = "https://github.com/quarto-dev/quarto-cli/releases/download/v1.0.38/quarto-1.0.38-linux-amd64.tar.gz";
+    sha256 = "sha256-bqEGRX6ihyrgmIxqkV9jr/2QfWM5LWnsjoGpTqK9qOc=";
+  };
+
+  buildInputs = [
+      pandoc
+      deno
+      esbuild
+  ];
+
+  installPhase = ''
```

Comment:
Add the missing pre/post `runHook`s: https://nixos.org/manual/nixpkgs/stable/#sec-stdenv-phases

Reply:
Solved in https://github.com/NixOS/nixpkgs/pull/186697/commits/ec9625cf7b85666835c5b59a3d9dd0cdbd009bc7